app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelTop.tsx

```diff
@@ -35,7 +35,10 @@ const ConnectionPanelTop = ({ connectedState, ipAddress, name }: ConnectionPanel
 					<StatusLabel text={connectedState} color={textColor} backgroundColor={backgroundColor} />
 				</div>
 				<div>
-					<p className={cx(ipAddressStyle)}>IP address: {ipAddress}</p>
+					{
+						ipAddress &&
```

Comment:
We won't have this data for everyone so rendering nothing if it's not available.